shifthappens/tasks/ccc/ccc_utils.py

```diff
@@ -266,15 +269,22 @@ def generate_dataset(self):
                 generated_subset = ApplyTransforms(self.data_dir, n1, n2, s1, s2, self.subset_size, self.target_dir)
                 dset2lmdb(generated_subset, path)
 
+            test_transform = tv_transforms.Compose(
+                [
+                    tv_transforms.ToTensor(),
+                    tv_transforms.Lambda(lambda x: x.permute(1, 2, 0)),
```

Comment:
Why is this needed?

Reply:
Here we just put channels into the last dimension.  Later we undone it in the model's part [transpose](https://github.com/shift-happens-benchmark/icml-2022/blob/main/shifthappens/models/torchvision.py#L26).

Tbh I don't remember why we are doing it this way :/. But here, we need it to have the correct format later in the model's API.

Reply:
Great, thanks